src/components/PopupConfirm.js

```diff
@@ -0,0 +1,29 @@
+import Popup from "./Popup.js";
+
+export default class PopupConfirm extends Popup {
+  constructor(popupSelector) {
+    super({ popupSelector });
+    this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._submitButton = this._popupForm.querySelector(".modal__button");
+  }
+
+  setEventListeners() {
+    this._popupForm.addEventListener("submit", (e) => {
+      e.preventDefault();
+      this._handleFormSubmit();
+    });
+    super.setEventListeners();
+  }
+
+  setConfirmCall(handleFormSubmit) {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.